Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Build stage
+ARG PYTHON_VERSION=3.9
+FROM python:${PYTHON_VERSION}-slim AS build
+WORKDIR /app
+
+COPY . .
+
+# Runtime stage
+FROM python:${PYTHON_VERSION}-slim
```

Comment:
Use your first stage as image for second. For example,
`FROM build-stage AS final-stage`